pytorch_lightning/trainer/trainer.py

```diff
@@ -818,6 +823,62 @@ def slurm_job_id(self) -> int:
             job_id = None
         return job_id
 
+    @staticmethod
+    def get_default_args():
+        return vars(Trainer())
```

Comment:
```suggestion
    @classmethod
    def get_default_args(cls):
        return vars(cls())
```
in case of inherited objects...